config/interface/output_element_series/use_of_final_demand_in_industry_other_metals.yml

```diff
@@ -47,15 +59,15 @@
   dependent_on: 
   output_element_key: use_of_final_demand_in_industry_other_metals
   key: electricity_use_of_final_demand_in_industry_other_metals
-- label: heat
-  color: "#8B0000"
```

Comment:
Why is heat removed here? There is a `industry_final_demand_for_metal_other_metals_steam_hot_water` node in the other metals sector.